test/test.js

```diff
@@ -169,7 +169,7 @@ describe('sendEvent', () => {
   it('should call _gaTransform if this.tid is set', () => {
       beforeWebExt(global);
       const m = new Metrics({id: '@my-addon', version: '1.0.2', uid: '12345', tid: 'UA-49796218-47'});
-      const clientStub = sinon.stub(m, '_sendToClient');
+      // const clientStub = sinon.stub(m, '_sendToClient');
```

Comment:
The send methods are designed to log errors, not throw them, so you can't rely on that to detect if something's going wrong. I think it's good practice to fully harness the methods, rather than let random stuff go wrong (in this case, trying to load Services.jsm will silently fail, I think).